pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -187,48 +208,45 @@ func (s *Service) HandleOutbound(msg *service.DIDCommMsg, destination *service.D
 }
 
 func (s *Service) handle(msg *message) error {
-	logger.Infof("entered into private handle didcomm message: %s ", msg)
+	logger.Infof("entered into private handle didcomm message: %s ", msg.ThreadID)
```

Comment:
update the msg as this is changed to threadID (eg "message with threadID").

Reply:
added